pkg/mon/mocks/factory.go

```diff
@@ -38,38 +38,32 @@ func NewLoggerMockedUntilLevel(level string) *Logger {
 		panic(fmt.Errorf("failed to find level to mock: %s", level))
 	}
 
-	mockLoggerMethod(logger, "Debug", "Debugf", mon.Debug, levelIndex >= levelMap[mon.Debug])
-	mockLoggerMethod(logger, "Info", "Infof", mon.Info, levelIndex >= levelMap[mon.Info])
-	mockLoggerMethod(logger, "Warn", "Warnf", mon.Warn, levelIndex >= levelMap[mon.Warn])
-	mockLoggerMethod(logger, "Error", "Errorf", mon.Error, levelIndex >= levelMap[mon.Error])
+	mockLoggerMethod(logger, "Debug", mon.Debug, levelIndex >= levelMap[mon.Debug])
+	mockLoggerMethod(logger, "Info", mon.Info, levelIndex >= levelMap[mon.Info])
+	mockLoggerMethod(logger, "Warn", mon.Warn, levelIndex >= levelMap[mon.Warn])
+	mockLoggerMethod(logger, "Error", mon.Error, levelIndex >= levelMap[mon.Error])
 
 	return logger
 }
 
 func inspectLogFunction(level string, allowed bool) func(args mock.Arguments) {
 	return func(args mock.Arguments) {
 		// ensure we use formatting markers exactly when using a method with formatting
```

Comment:
I think we can delete this comment, it is no longer valid

Reply:
done